packages/grafana-runtime/src/index.ts

```diff
@@ -7,7 +7,7 @@ export * from './services';
 export * from './config';
 export * from './types';
 export { loadPluginCss, SystemJS, PluginCssOptions } from './utils/plugin';
-export { reportMetaAnalytics, reportInteraction, reportPageview } from './utils/analytics';
+export { reportMetaAnalytics, reportInteraction, reportPageview, reportExperimentview } from './utils/analytics';
```

Comment:
That's an unusual selling, what about:
```suggestion
export { reportMetaAnalytics, reportInteraction, reportPageview, reportExperimentView } from './utils/analytics';
```

Reply:
I agree, I just wanted to keep it consistent with `reportPageview`.